primer/brand
# Security and dependency updates

## Summary


Part of https://github.com/github/primer/issues/2297

Updates all monorepo depenencies to their latest patch/minor versions and fixes various critical security vulnerabilities, as reported by Dependabot. This should close most - if not all - open Dependabot pull requests.

This is step one of a wider effort to bring the repo up-to-date and reduce the number of security alerts.

## List of notable changes:





- Before: 75 outdated dependencies
- After: 45 outdated dependencies (all major)

- Typescript has been upgraded from 4.6 to 4.9, which includes stricter typing around ReactElements. 
- Fixes a11y violation around tabIndex on SubdomainNavBar

## What should reviewers focus on?

- Checking builds pass CI and integration checks
- Review the [documentation](https://primer-2b869135c5-26139705.drafts.github.io/) and verify it still works as expected
- [Review Storybook](https://primer-2b869135c5-26139705.drafts.github.io/storybook)

## Steps to test:



1. Open the preview deployment links here
2. Navigate between all (or most) stories and documentation pages
3. Verify that components appear the way they should


## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

<img width="998" alt="Screenshot 2023-05-25 at 18 38 30" src="https://github.com/primer/brand/assets/13340707/bd88f7e5-ba74-4805-a00e-3136e5b4fcd2">

 </td>
<td valign="top">

<img width="1097" alt="Screenshot 2023-05-26 at 09 24 06" src="https://github.com/primer/brand/assets/13340707/31a5f0a3-61a9-4c2e-bae6-2c62417adc81">

</td>
</tr>
</table>
